### PR TITLE
Include kaytus.ksmanage collection in ansible 9

### PR DESCRIPTION
**Pull request by Andersson007**
Include kaytus.ksmanage collection in ansible 9

https://forum.ansible.com/t/default-date-2024-04-25-collection-inclusion-include-kaytus-ksmanage-in-the-ansible-community-package/5016

### CODE REVIEW

**Comment by Andersson007** [comment]
i followed https://github.com/ansible-community/ansible-build-data?tab=readme-ov-file#adding-a-new-collection, what's missed here?

File: 9/ansible-9.build [hunk 1]
```diff
@@ -76,6 +76,7 @@ inspur.ispim: >=2.1.0,<3.0.0
 inspur.sm: >=2.3.0,<3.0.0
 junipernetworks.junos: >=5.3.0,<6.0.0
 kubernetes.core: >=2.4.0,<3.0.0
+kaytus.ksmanage: >=1.2.1,<2.0.0
```
Comment (felixfontein) [inline 9/ansible-9.build:79]: I think you need to add this to 10/ansible-10.build as well.

(Also please add this one line up, before kubernetes.core.)

File: 10/ansible.in [hunk 2]
```diff
@@ -69,6 +69,7 @@ inspur.ispim
 inspur.sm
 junipernetworks.junos
 kubernetes.core
+kaytus.ksmanage
```
Comment (felixfontein) [inline 10/ansible.in:72]: Please add `kaytus.ksmanage` before `kubernetes.core`, for alphabetical sorting :) Same in the other files.

File: 9/ansible-9.build (hunk 1 above)
Comment (Andersson007) [reply 9/ansible-9.build:78]: thanks! it's not reflected in README in the adding collections to the next major release section

File: 10/ansible.in (hunk 2 above)
Comment (Andersson007) [reply 10/ansible.in:71]: Oh, no idea why i looked at `y` instead of instead of `a`:) Fixed, PTAL

**Comment by Andersson007** [comment]
Fixed, ready for review, thanks

File: 9/ansible-9.build (hunk 1 above)
Comment (felixfontein) [reply 9/ansible-9.build:78]: Yes, it should be added.

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@Andersson007 thanks!

**Pull request merged**